### PR TITLE
added first story and quizSummary

### LINKED ISSUE #43: Implement first story

### PR DESCRIPTION
**Pull request by emdevs**
In this branch I've also created a quizSummary constructor function, to add support for multiple quiz questions. Let me know if I should move this function to a different folder/rename things etc. 

closes #43

### CODE REVIEW

**Review by Tudwall — APPROVED** [review]
Looks good to me! :)

File: src/js/quiz-factory.js [hunk 1]
```diff
@@ -10,4 +10,25 @@ const Quiz = (question, choices, answer) => {
...
 };
 
-export default Quiz;
+const QuizSummary = (quizArray) => {
+  let passed = false;
+  let quizzes = [];
+
+  for (let i=0; i < quizArray.length; i++) {
+    let quiz = Quiz(quizArray[i].question, quizArray[i].choices, quizArray[i].answer);
+    quizzes.append(quiz);
```
Comment (DavidMatalik) [inline src/js/quiz-factory.js]: Do you mean `quizzes.push(quiz)`?

File: src/js/stories/1/dnd.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+//Drag and Drop questions
```
Comment (DavidMatalik) [inline src/js/stories/1/dnd.js:1]: For my understanding: Are these questions displayed in the drag and drop screen? And the player has the task to answer these questions with dragging the right thing?

**Review by DavidMatalik — COMMENTED** [review]
Looks also good to me =) No moving/renaming necessary from my side 

What you could do as well in this PR: make the assets of the first story work with the game logic. The quiz logic isn't implemented yet, but images and story text could be implemented easily I think. 

Or we do this in an extra PR and game-logic expert @mdlugosz1 takes care of it.

**Comment by mdlugosz1** [comment]
> 
> 
> Looks also good to me =) No moving/renaming necessary from my side
> 
> What you could do as well in this PR: make the assets of the first story work with the game logic. The quiz logic isn't implemented yet, but images and story text could be implemented easily I think.
> 
> Or we do this in an extra PR and game-logic expert @mdlugosz1 takes care of it.

It's entirely up to @emdevs, i will adjust.

**Review by mdlugosz1 — APPROVED** [review]

File: src/js/quiz-factory.js (hunk 1 above)
Comment (emdevs) [reply src/js/quiz-factory.js]: whoops, yeah that's meant to be push lol. ill fix that.

File: src/js/stories/1/dnd.js (hunk 2 above)
Comment (emdevs) [reply src/js/stories/1/dnd.js:1]: Yup, that's right

**Review by DavidMatalik — APPROVED** [review]

**Pull request merged**